api/src/pcapi/routes/backoffice_v3/templates/components/forms/date_field.html

```diff
@@ -3,8 +3,10 @@
   <input type="date"
          id="{{ date_field_id }}"
          name="{{ field.name }}"
+         data-original-name="{{ field.name }}"
          class="form-control value-element-form"
          value="{{ field.data if field.data else "" }}" />
   <label class="label-element-form"
+         data-original-name="{{ field.name }}"
```

Comment:
Ok why not, but, est-ce qu'on en a vraiment besoin 2 fois ? on pourrait le poser depuis JS, via son scope ?